Signal/src/call/CallService.swift

```diff
@@ -167,11 +167,13 @@ fileprivate let timeoutSeconds = 60
 
         return getIceServers().then(on: CallService.signalingQueue) { iceServers -> Promise<HardenedRTCSessionDescription> in
             Logger.debug("\(self.TAG) got ice servers:\(iceServers)")
-            let peerConnectionClient = PeerConnectionClient(iceServers: iceServers, peerConnectionDelegate: self)
-            self.peerConnectionClient = peerConnectionClient
+            let peerConnectionClient = PeerConnectionClient(iceServers: iceServers, delegate: self)
+
+            // When placing an outgoing call, it's our responsibility to create the DataChannel. Recipient will not have
+            // to do this explicitly.
+            peerConnectionClient.createSignalingDataChannel()
 
-            // When calling, it's our responsibility to create the DataChannel. Receivers will not have to do this explicitly.
-            self.peerConnectionClient!.createSignalingDataChannel(delegate: self)
+            self.peerConnectionClient = peerConnectionClient
```

Comment:
> So, there's only one peerConnectionClient at a time.
right

> If a client has an ongoing call and receives another WebRTC call, that new call's peerConnectionClient isn't set up until the user hangs up the first by confirming that they want to switch?

Well... conceivably, but in the current iteration when you are already in a call when receiving another call, you send a "BUSY" message. =/